wgpu-hal/src/gles/device.rs

```diff
@@ -87,6 +87,7 @@ impl super::Device {
         };
 
         let raw = gl.create_shader(target).unwrap();
+        #[cfg(not(target_arch = "wasm32"))]
         if gl.supports_debug() {
```

Comment:
is this method not exposed to wasm at all?

Reply:
The `supports_debug()` function technically works and always returns `false`, as it should, but the issue is actually the `mem::transmut(raw)` won't compile on WASM because the `raw` type is something like `WebShaderKey`, which can't be transmuted to a `u32` because of the size difference.